README.md

```diff
@@ -349,6 +349,64 @@ const home = (state, actions) => {
 app.addActions({votes: voteActions})
 ```
 
+### `app.addListener(listener)`
```

Comment:
New Documentation for `addListener` function.
The example is pretty much copy-paste from `addActions`